app/Http/Controllers/ImportsResultsController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ImportMeta;
+use App\Services\ImportResults;
+
+class ImportsResultsController extends Controller
+{
+    public function showResultsCsv(int $import_id, ImportResults $results)
+    {
+        $import = ImportMeta::findOrFail($import_id);
+
+        $filename = strtr(config('imports.results.filename_template'), [
+            ':datetime' => now()->format('Y-m-dTH:i:s')
+        ]);
```

Comment:
```suggestion
            ':id' => $import_id
            ':datetime' => now()->format('Y-m-dTH:i:s')
        ]);
```

In accordance with the file name template change suggested in the comment above.